packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -61,7 +62,7 @@ export const FinalFormSelect = <T,>({
     children,
     required,
     ...rest
-}: FinalFormSelectProps<T> & Partial<AsyncOptionsProps<T>> & Omit<SelectProps, "input" | "endAdornment">) => {
+}: FinalFormSelectProps<T> & Partial<AsyncOptionsProps<T>> & Omit<SelectProps, "input" | "endAdornment" | "error">) => {
```

Comment:
As mui already has an `error` prop, maybe it's better to name it more specific? Like `loadingError`?